workshop/learn.ftd

```diff
@@ -1,34 +1,45 @@
 -- ds.page-with-no-right-sidebar: Learning Resources
 
-Here are some important links which helps you to create your `fastn` websites
-and dynamic apps. 
-
--- ds.h1: Create website
-
-- Nandini’s [blog post](/writer/)
-- [acme case study](/acme/) 
-- [fastn template](https://github.com/fastn-stack/fastn-template) 
-- Learn [markdown](/markdown/-/frontend/) 
-- [workshop](https://github.com/fastn-stack/workshop)
-
--- ds.h1: Learn how to customise design of your `fastn` site
-
-- how to add [color scheme](/color-scheme/) 
-- how to add [typography](/typography/)
-- [figma tokens with fastn](/figma/)
-- how to [create font](/create-font-package/)
-
--- ds.h1: Learn how to build reusable components with `fastn` 
-
-- [expander](/expander/)
-- workshop 🚧
-
--- ds.h1: Learn how to build full stack application
-
-- [full stack](/backend/) 
-- [todo app](https://github.com/fastn-community/todo-app)
-- [basics of http processor and data modelling](/country-details/basics/)
-- [building dynamic country list page](/country-list/)
-- workshop 🚧
+Discover a wealth of resources to help you master `fastn` web development, from
+creating websites to building dynamic apps.
+
+-- ds.h1: Learn How to Create a Website
+
+- **Nandini’s Blog Post:** Get started with creating websites using `fastn`.
```

Comment:
Make this "A content writer's experience with fastn".

Reply:
Also never use "Read it here" or "click here" etc as link target. Google attaches text of the link. If someone search "read it here", Google search will show the `/writer/` page. If you have have used google you will know you never search for words like "here" or "click here" etc. What you search for should be in the text of the link.

- [A content writer's experience with fastn](/writer/)